.github/workflows/snapshot-release.yml

```
@@ -0,0 +1,145 @@
+name: Snapshot Release
+
+on:
+  issue_comment:
+    types:
+      - created
+
+jobs:
+  release_next:
+    name: release:next
+    runs-on: ubuntu-latest
+    # Permissions necessary for Changesets to push a new branch and open PRs
+    # (for automated Version Packages PRs), and request the JWT for provenance.
+    # More info: https://docs.github.com/en/actions/deployment/security-hardening-your-deployments/about-security-hardening-with-openid-connect#adding-permissions-settings
+    permissions:
+      contents: write
+      pull-requests: write
+      id-token: write
+    if: |
+      github.repository == 'siemens/ix' &&
+      github.event.issue.pull_request &&
+      (
+        github.event.sender.login == 'danielleroux' ||
+        github.event.sender.login == 'danielleroux'
```

Required for me to test this, correct?
```suggestion
        github.event.sender.login == 'nuke-ellington'
```